common/src/main/java/org/apache/cassandra/diff/RetryStrategy.java

```diff
@@ -16,12 +17,28 @@
     protected abstract boolean shouldRetry();
 
     public final <T> T retry(Callable<T> retryable) throws Exception {
+        return retryIfNot(retryable, FakeException.class);
```

Comment:
why do we need FakeException here? Looks like just calling it without excluded exception would provide the same result